lib/flags.ml

```diff
@@ -48,9 +48,9 @@ let profile = false
 let ide_slave = ref false
 let ideslave_coqtop_flags = ref None
 
-let raw_print = ref false
-
-let univ_print = ref false
+let univ_print_explanations = ref true
```

Comment:
Why is this option not in the record?

Reply:
Because it doesn't deal with term printing, just error messages.